cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -8,9 +8,15 @@ akka.management {
   http.route-providers += "akka.management.cluster.bootstrap.ClusterBootstrap$"
 
   cluster.bootstrap {
-    # When enabled (default), a new cluster will be formed via the bootstrapping mechanism. This can be disabled to only
-    # allow a node to join already existing clusters.
-    form-new-cluster = on
+
+    # Cluster Bootstrap will always attempt to join an existing cluster if possible. However
+    # if no contact points are found during service discovery, a new cluster will be formed.
+    # Setting `new-cluster-enabled=off` after an initial cluster has formed is recommended,
+    # accompanied by a re-deploy. Defaults to 'on'.
+    new-cluster-enabled = on
+
+    # The timeout applied to the bootstrap process.
+    timeout = 1 d
```

Comment:
Does anyone know why this was set to a day in the first place? It seems very long.

Reply:
+1 I can not think of a reason we would want 1 day

Reply:
the code has a comment
```
// the boot timeout not really meant to be exceeded
```

it's the timeout of a local `ask`

I guess it makes sense to have some (shorter) timeout and if bootstrap isn't successful it should run CoordinatedShutdown (stop the process, and let K8 start over again). Right now it's only logging a warning if the timeout is exceeded and nothing more, which isn't a good solution. Perhaps create a separate issue for this.

Reply:
@chbatey I like that idea because it solves the reason I added it - not having something like that hard coded and also keeps it simple. May create another ticket per @patriknw's thought.

Reply:
The shorter timeout achieves what can already be done with `akka.cluster.shutdown-after-unsuccessful-join-seed-nodes` 

The restart the process was discussed in https://github.com/akka/akka-management/issues/356 and was convinced by @patriknw that it wasn't worth it as in nearly all environments if we shutdown we'll be restarted.

I think for now we should remove the timeout, make it a `tell` and don't log a warning. Bootstrap logs a lot if it can't form a cluster so the user won't be short of logs describing why a cluster hasn't formed